Increasing hash key length

subhash-arabhi opened the pull request:
### Issue
Hash key length is different in different sections of project. We are increasing two of them to 64
### Changes
Increased hash key length to 64 in below projects
Bootstrap/org.netbeans.CLIHandler and 
Gradle-Projects/org.netbeans.modules.gradle.ProjectTrust

matthiasblaesing reviewed (commented):
Please ensure the commits are grouped into atomic changes. There is no squash-on-merge in the NetBeans repository, as github side merges are unreliable regarding the author information.

What is missing in the commit message and the PR is the motivation. The commit message currently only gives the "what", but that is already answered by the diff, the "why" is the interesting part.

If I get it correctly, it was determined, that 14-18 bytes are not enough randomness. And the keys are used to verify local connections from NB cli and gradle.

subhash-arabhi commented:
> Please ensure the commits are grouped into atomic changes. There is no squash-on-merge in the NetBeans repository, as github side merges are unreliable regarding the author information.
> 
> What is missing in the commit message and the PR is the motivation. The commit message currently only gives the "what", but that is already answered by the diff, the "why" is the interesting part.
> 
> If I get it correctly, it was determined, that 14-18 bytes are not enough randomness. And the keys are used to verify local connections from NB cli and gradle.

@matthiasblaesing Yes, we want to increase the randomness. I added this in the description and squashed the commits. Please approve the PR

matthiasblaesing commented:
@subhash-arabhi please update the commit message. The commit message should explain _why_ a change was done. What is more the summary line should give a meaningful summary. Reading the current summary I wonder which hash key lengths?

This still leaves the question: why is 64 right and 16 wrong. How did you determine that? Another question: is this just a random value at runtime or is it stored somewhere? If it is stored, why are the changed lengths not problematic?
